hubble/src/indexer/tm/fetcher_client.rs

```diff
@@ -221,7 +224,7 @@ impl TmFetcherClient {
 
         let pg_transactions = transactions_response
             .into_iter()
-            .filter(|tx| tx.tx_result.code.is_ok())
+            .filter(|tx| tx.tx_result.code == 0) // 0 == OK
```

Comment:
perhaps use a const: `CODE_OK = 0`, although that is a nit tbh.

Reply:
fixed in f40e13a0c013fb1bd8061e281ae6cd5dda72f610